server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -100,7 +101,9 @@ export async function runWithRetry<T>(
 			metric.setProperty("maxRetries", maxRetries);
 			metric.setProperty("retryAfterMs", retryAfterMs);
 			if (success) {
-				metric.success("runWithRetry succeeded");
+				if (!shouldIgnoreInitialSuccess || retryCount > 0) {
+					metric.success("runWithRetry succeeded");
```

Comment:
One thing to note here is that the Lumber metric will potentially never be completed. I think it does not make much difference now, since we do not do much cleanup when emitting/completing the metric. But in case we even do that, to perhaps optimize things like garbage collection, that might not apply here. But maybe my point is not even that accurate, since Node.js might be garbage collecting its own things in different ways than I thought of

Reply:
This is a good point. Agreed this is not a issue now, but it might be a bug or blocker if in the future we want to do cleanup when emitting the metric. So maybe we can change it to verbose to avoid the potential future issue.